actionview/lib/action_view/helpers/cache_helper.rb

```diff
@@ -198,10 +198,12 @@ def fragment_cache_key(key)
 
     private
 
-      def fragment_name_with_digest(name) #:nodoc:
-        if @virtual_path
+      def fragment_name_with_digest(name, virtual_path: nil) #:nodoc:
```

Comment:
`virtual_path` doesn't need to be a keyword argument, when we're passing it straight down from `cache_fragment_name`.